### DISCUSSION TITLE
mac silicon install issue

### DISCUSSION THREAD

**Discussion by rzuhi** [talmolab/sleap#1341] [Help!]
When running conda env create -f environment_apple_silicon.yml, I get an error:
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× pip subprocess to install build dependencies did not run successfully.
│ exit code: 1
╰─> See above for output.
CondaEnvException: Pip failed

**Comment by roomrys** [comment]
Hi @rzuhi,

I noticed the discussion is marked as closed. Did you solve your problem? If so, what did you do to solve it?

Thanks,
Liezl
